scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1668,15 +1670,21 @@ private void insertVisualization(List<JobInfo> jobsInfo) {
     private void insertSignals(List<JobInfo> jobsInfo) {
         jobsInfo.stream().filter(jobInfo -> {
             try {
+                TaskId taskId = getNoTaskId(jobInfo.getJobId().value());
                 return publicStore.keySet(SIGNAL_ORIGINATOR,
-                                          SIGNAL_TASK_ID,
+                                          taskId,
                                           signalsChannel + jobInfo.getJobId().value()) != null;
             } catch (InvalidChannelException e) {
             }
             return false;
         }).forEach(jobInfo -> insertJobSignals(jobInfo));
     }
 
+    private TaskId getNoTaskId(String value) {
```

Comment:
the parameter should be named jobId

the method name should be createFakeTaskId